components/avatar/avatarPage.html

```diff
@@ -56,6 +56,20 @@ <h3 class="header__title">
         <section class="container__sidebar">
             <!-- Drawer Component Regular -->
             <div class="drawer">
+
+                <!-- Getting Started Starts -->
+                <div class="drawer__heading">
+                    <h3 class="drawer__title">Getting Started</h3>
+                </div>
+                <div class="drawer__content-holder" style="margin-bottom: 2rem;">
```

Comment:
Any specific reason for the inline style?
Apart from that code looks good to me.

Reply:
+1

Reply:
Cool then, you can go ahead.